cnf-certification-test/platform/suite.go

```diff
@@ -246,3 +250,14 @@ func testIsSELinuxEnforcing(env *provider.TestEnvironment) {
 		ginkgo.Fail(fmt.Sprintf("Failed because %d nodes are not running selinux", nodesFailed))
 	}
 }
+
+func testUnalteredBootParams(env *provider.TestEnvironment) {
+	failedContainers := []string{}
+	for _, cut := range env.Containers {
```

Comment:
What about iterating directly through the debug pods? As the test seems to be checking just node params, we could avoid repeating the test for all the containers that share the same node.

Reply:
we do not have a structure to hold the list of container per node/debug pods. This is achieving the same